app/views/patients/update.js.erb

```diff
@@ -36,3 +36,14 @@ $('#change_log').html(
   // clear out fulfillment form
   $('#fulfillment').empty();
 <% end %>
+
+// flash success
+if ($( "#flash" ).length == 0) {
```

Comment:
This conditional handles the case if the user deletes the flash notification, then another will pop up right after.